packages/zoe/src/objArrayConversion.js

```diff
@@ -18,3 +18,50 @@ export const objToArray = (obj, keywords) => {
   );
   return keywords.map(keyword => obj[keyword]);
 };
+
+export const assertSubset = (whole, part) => {
+  part.forEach(key => {
+    assert.typeof(key, 'string');
+    assert(whole.includes(key), details`key ${key} was not an expected key`);
+  });
+};
+
+// Keywords must equal the keys of obj
+export const filterObjToArray = (obj, keywords) => {
```

Comment:
`filter...` seems like the wrong verb when no elements will be removed. This ensures that all the keys are the same, so nothing will be lost.

Reply:
Good point